frontend/src/App.js

```diff
@@ -58,6 +62,31 @@ const theme = createMuiTheme({
 });
 
 function App() {
+  const [isAuthenticated, userHasAuthenticated] = useState(false);
+
+  async function Auth() {
+    try {
+      await ky.get("/api/get-access-token");
+      userHasAuthenticated(true);
+      isAuthenticated(true);
```

Comment:
This is probably unintended or I'm missing something. This value should only be a boolean.